neo/Network/P2P/Payloads/InvocationTransaction.cs

```diff
@@ -12,9 +12,7 @@ public class InvocationTransaction : Transaction
         public byte[] Script;
         public Fixed8 Gas;
 
-        public override int Size => base.Size
-            + Script.GetVarSize()           //Script
-            + Version >= 1 ? Gas.Size : 0;  //Gas
+        public override int Size => base.Size + Script.GetVarSize() + (Version >= 1 ? Gas.Size : 0);
```

Comment:
better to leave it on multiple lines.

Reply:
my Atom editor does not consider `+ Script.GetVarSize()           //Script` as a comment... can you believe this? So I spent some time thinking line breaking was an issue too....

Reply:
Fixed!